### PR TITLE
Add some #SquirrelTruth all up in here

### PR DESCRIPTION
**Pull request by tianon**

### CODE REVIEW

File: moxie/facts.py [hunk 1]
```diff
@@ -114,4 +114,8 @@ def get_printable_fact():
     "Error. Error. Error. Fact not found.",
     "Fact not found.",
     "Warning, sphere corruption at twenty-- rats cannot throw up.",
+
+    "Statistically speaking, at least one 'person' on this train is actually 7 squirrels wearing a human suit. Don't be a victim.",
+    "Squirrels live in trees. Bees live in trees. Are squirrels actually killer bees...in disguise???",
+    "Squirrels have never, ever wished you a Habby Birthday. Like, not even once. Probably because squirrels are inconsiderate, selfish monsters.",
```
Comment (paultag) [inline moxie/facts.py]: `Habby Birthday` :)

**Comment by tianon** [comment]
Ping :wink:

**Comment by paultag** [comment]
duh right thanks

**Comment by tianon** [comment]
MOAR SQUIRRELS CAPTAIN!

**Pull request merged**